ivc/src/poseidon_8_56_5_3_2/bn254/mod.rs

```diff
@@ -1,10 +1,18 @@
 //! Poseidon parameters that can be used by [crate::poseidon_8_56_5_3_2] over
-//! the scalar field of BN254
+//! the scalar field of BN254, for a security level of 128 bits.
 
-use ark_bn254::Fr as Fp;
+use crate::poseidon_8_56_5_3_2::columns::PoseidonColumn;
+use ark_bn254::Fr;
 use mina_poseidon::{constants::SpongeConstants, poseidon::ArithmeticSpongeParams};
 use once_cell::sync::Lazy;
 
+pub const STATE_SIZE: usize = 3;
+pub const NB_FULL_ROUND: usize = 8;
+pub const NB_PARTIAL_ROUND: usize = 56;
+pub const NB_TOTAL_ROUND: usize = NB_FULL_ROUND + NB_PARTIAL_ROUND;
+
+pub type Column = PoseidonColumn<STATE_SIZE, NB_FULL_ROUND, NB_PARTIAL_ROUND>;
```

Comment:
This could be redefined `PoseidonColumn` maybe... I find generic local-scope names always very confusing.